tasks/copy_to_glacier/copy_to_glacier.py

```diff
@@ -171,13 +186,25 @@ def task(event: Dict[str, Union[List[str], Dict]], context: object) -> Dict[str,
     copied_file_urls = []
 
     # initiate empty SQS body dict
-    sqs_body = {"provider": {}, "collection": {}, "granule": {}}
-    sqs_body["provider"]["name"] = config.get("providerName", None)
-    sqs_body["provider"]["providerId"] = config["providerId"]
-    sqs_body["collection"]["shortname"] = config["collectionShortname"]
-    sqs_body["collection"]["version"] = config["collectionVersion"]
-    # Cumulus currently creates collectionId by concatenating shortname + ___ + version
-    # See https://github.com/nasa/cumulus-dashboard/blob/18a278ee5a1ac5181ec035b3df0665ef5acadcb0/app/src/js/utils/format.js#L342
+    sqs_body = {
+        "provider": {}, "collection": {}, "granule": {}
+        }
+    sqs_body["provider"]["name"] = config.get(
```

Comment:
These still feel weird to me. Most of these don't even break the 99 character limit.

If Flake insists that they cannot remain, I'd recommend
sqs_body["key"]["key"] = \
    config.get("key")
so we don't split the config access to 3 lines.
Recommend similar changes to config["key"] usages below.

Reply:
yes it is weird. 

Not sure why it was giving me error on Friday. I reverted the changes and didn't get any error.